### DISCUSSION TITLE
Support config line ending in config.toml?

### DISCUSSION THREAD

**Discussion by wsw0108** [helix-editor/helix#3945] [Q&A]
When open new file with `:o file` under Windows, the line ending is `CRLF` which is not I wanted.
Does there has a way to specify line ending in config file?

**Comment by go2null** [comment]
Would be great to also allow setting the default EOL.
* I'd like to use `LF` even when I'm working in `MSYS2`.

**Reply by CptPotato** [reply]
fyi, there's an open PR that addresses this: #5621

**Reply by go2null** [reply]
PR merged
* https://github.com/helix-editor/helix/issues/4378+
* https://github.com/helix-editor/helix/pull/5621+

**Reply by go2null** [reply]
NOTE: The new setting is `default-line-ending` and it is the `[editor]` [section](https://docs.helix-editor.com/configuration.html#editor).
